helpdb/sid-registers.json

```diff
@@ -0,0 +1,175 @@
+{
+    "d400": {
+        "descr": "Voice #1 frequency. Write-only.",
+        "name": "",
+        "url": ""
+    },
+
+    "d401": {
+        "descr": "Voice #1 frequency. Write-only.",
+        "name": "",
+        "url": ""
+    },
+
+    "d402": {
+        "descr": "Voice #1 pulse width. Write-only.",
+        "name": "",
+        "url": ""
+    },
+
+    "d403": {
+        "descr": "Voice #1 pulse width. Write-only.",
+        "name": "",
+        "url": ""
+    },
+
+    "d404": {
+        "descr": "Voice #1 control register. Write-only",
+        "name": "",
+        "url": ""
+    },
+
+    "d405": {
+        "descr": "Voice #1 Attack and Decay length. Write-only",
+        "name": "",
+        "url": ""
+    },
+
+    "d406": {
+        "descr": "Voice #1 Sustain volume and Release length. Write-only",
+        "name": "",
+        "url": ""
+    },
+
+    "d407": {
```

Comment:
Lots of sprite stuff in the SID-registers ;)

Reply:
Pushed updated version of the helpfile. I don't know if it'll automatically update the pull request. Reopen and kick me if not!